app/notification.go

```diff
@@ -657,6 +753,44 @@ func (a *App) getMentionKeywordsInChannel(profiles map[string]*model.User, allow
 	return keywords
 }
 
+// insertGroupMentions adds Mentions for users that are part of the given group members and channel channel, adds group members that are not in the channel to OtherPotentialMentions
```

Comment:
maybe rephrase to say "for users that are members of the given group", etc.

Reply:
the wording on this was horrible - ill fix it